js/src/jit/JitSpewer.cpp

```diff
@@ -629,6 +629,8 @@ void jit::DisableChannel(JitSpewChannel channel) {
   LoggingBits &= ~(uint64_t(1) << uint32_t(channel));
 }
 
+#endif /* JS_JITSPEW */
+
 const char* js::jit::ValTypeToString(JSValueType type) {
```

Comment:
What do you think to wrapping changes like this one within a `if defined(JS_JITSPEW) || defined(ENABLE_JS_AOT_ICS)`?

Reply:
Done!